modules/common/createValidAzResourceName.bicep

```diff
@@ -21,6 +21,7 @@ param location string
   'webApp' // Web App
   'plan' // App Service Plan
   'appi' // Application Insights
+  'law' // Log Analytics Workspace
```

Comment:
You also need to update the `Defs` object on line 46, otherwise the name cannot be successfully generated.